### PR TITLE
Use view binding in VideoPlayer.

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [x] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Use view binding in `VideoPlayer` and `VideoPlayerImpl`.

#### APK testing 

[debug.zip](https://github.com/TeamNewPipe/NewPipe/files/5732811/debug.zip)

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java [hunk 1]
```diff
@@ -178,54 +153,37 @@ private static boolean containsCaseInsensitive(final List<String> list, final St
...
-        this.bottomControlsRoot = view.findViewById(R.id.bottomControls);
-        this.topControlsRoot = view.findViewById(R.id.topControls);
-        this.qualityTextView = view.findViewById(R.id.qualityTextView);
-
-        this.subtitleView = view.findViewById(R.id.subtitleView);
+    public void initViews(@NonNull final PlayerBinding binding) {
+        playerBinding = binding;
+        playbackSeekBar = (SeekBar) binding.playbackSeekBar;
+        qualityTextView = (TextView) binding.qualityTextView;
+        playbackSpeed = (TextView) binding.playbackSpeed;
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java:168]: Why did you keep these three variables?

**Review by Stypox — CHANGES_REQUESTED** [review]
I tested and it works, changes also look good, thank you :-D
Maybe you could rename `playerBinding` into just `binding`, so that there is less code to read. `closeOverlayBinding` is ok though, since it's a specific binding.

File: app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java (hunk 1 above)
Comment (Isira-Seneviratne) [reply app/src/main/java/org/schabi/newpipe/player/VideoPlayer.java:168]: I was getting compilation errors as those three views were being stored as `View` variables for some reason.

**Comment by Isira-Seneviratne** [comment]
> I tested and it works, changes also look good, thank you :-D
> Maybe you could rename `playerBinding` into just `binding`, so that there is less code to read. `closeOverlayBinding` is ok though, since it's a specific binding.

I chose the name to keep the bindings distinguishable, but I'll rename the binding.

**Review by Stypox — APPROVED** [review]
I rebased and tested. This works. Thank you again :-D
Now I'll have to rebase #5371, wish me luck ;-)

**Pull request merged**